spec/filters/syslog_pri_spec.rb

```diff
@@ -153,30 +117,19 @@
 
         it "syslog severity is notice" do
           subject.filter(event)
-          if ecs_compatibility?
-            expect(event.get("[log][syslog][severity][name]")).to eq("alert")
-          else
-            expect(event.get("syslog_severity")).to eq("alert")
-          end
+          expect(event.get(syslog_severity_name_field)).to eq("alert")
         end
 
         it "default syslog_facility is user-level" do
           subject.filter(event)
-          if ecs_compatibility?
-            expect(event.get("[log][syslog][facility][name]")).to eq("local1")
-            expect(event.get("[log][syslog][facility][code]")).to eq(17)
-          else
-            expect(event.get("syslog_facility")).to eq("local1")
-            expect(event.get("syslog_facility_code")).to eq(17)
-          end
+          expect(event.get(syslog_facility_name_field)).to eq("local1")
+          expect(event.get(syslog_facility_code_field)).to eq(17)
         end
       end
 
       context "when malformed messages arrive" do
         context "if syslog priority value is too high" do
           let(:syslog_pri) { 193 }
-          let(:syslog_facility_code_field) { ecs_compatibility? ? "[log][syslog][facility][code]" : "syslog_facility_code" }
-          let(:syslog_severity_code_field) { ecs_compatibility? ? "[log][syslog][severity][code]" : "syslog_severity_code" }
           let(:syslog_facility_label_field) { ecs_compatibility? ? "[log][syslog][facility][label]" : "syslog_facility" }
           let(:syslog_severity_label_field) { ecs_compatibility? ? "[log][syslog][severity][label]" : "syslog_severity" }
```

Comment:
we can delete these too?

```suggestion
```

Reply:
The ECS name `[log][syslog][facility][name]` is different from `[log][syslog][facility][label]` also if the non-ECS `syslog_facility` are the same.
So we can't delete

Reply:
Mmm the plugin code newer use  `[log][syslog][facility][label]` so why spec was working :thinking: